src/Core/Utility/Form/Control.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2020 ChannelWeb Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+
+namespace App\Core\Utility\Form;
+
+use Cake\Core\Configure;
+use Cake\Utility\Hash;
+use Cake\Utility\Inflector;
+
+class Control
```

Comment:
Good job! A class refactor was needed and class decomposition is fine.

* Con we add a comment line on each class to describe its purpose?
* why `\App\Core\Utility\Form` namespace and not `\App\Utility\Form` or `\App\Form` ?

Reply:
> Con we add a comment line on each class to describe its purpose?
class comments incoming soon ;)

> why \App\Core\Utility\Form namespace and not \App\Utility\Form or \App\Form ?
good point. namespace `\App\Form` could be fine. I'll apply it soon